src/ride/views.py

```diff
@@ -49,3 +50,17 @@ def post(self, request, id):
         except json.JSONDecodeError:
             data = request.data
         return UpdateRideController(request, data, self.serializer_class, id).process()
+
+
+class SearchView(generics.GenericAPIView):
+    serializer_class = RideSerializer
+    permission_classes = api_settings.CONSUMER_PERMISSIONS
+
+    def get(self, request):
+        """Search for a ride."""
+        try:
+            data = json.loads(request.body)
+        except json.JSONDecodeError:
```

Comment:
let's make a push to get rid of the try except and just load the json - I don't think we'll ever have a case where we have a request that is not a JSON.

Reply:
lol i took this from pear-django-backend; i don't think it hurts to have this reinforcement

Reply:
changed x4